agent/src/policy/policy.rs

```diff
@@ -185,7 +186,25 @@ impl Policy {
     ) {
         let (socket_extra_info_0, socket_extra_info_1) = socket_extra_infos;
 
-        match packet.lookup_key.direction {
+        let direction = match packet.lookup_key.direction {
```

Comment:
加注释：

We consider the direction (role) in socket_extra_info to be the ground truth because it is taken from the socket information of the OS. Both meta_packet and flow need to correct their direction based on this.